Expanded-info.md

```diff
@@ -0,0 +1,48 @@
+#PR for discussion
+
+## Problem description
+
+Today, if you use JSON generated from *.vspec information on whether a branch existed in *.vspec or comes from
+instance expansion (e.g. DriverSide, PassengerSide) is lost. This makes it difficult if you want to handle instantiation
+differently in your tool, i.e. you want a representation similar to what exist in *.vspec files.
+
+This PR prototypes two methods, and is intended for discussion
+
+## Alternative 1: Annotate that a branch is expanded
+
+In this PR an example for JSOn is shown, running as today:
+
+```
+vss-tools/vspec2json.py -I ./spec --uuid -u ./spec/units.yaml --json-pretty ./spec/VehicleSignalSpecification.vspec expand.json
+```
+
+...gives an extra output `expanded:true` for branches that has been expanded.
+
+I.e. branches like `Vehicle.Cabin.Seat.Row2.PassengerSide` will have `expanded:true`, while regular branches like
+`Vehicle.Cabin` and `Vehicle.Cabin.Seat` will not have any `expanded` attribute.
+
+## Alternative 1B: Keep original instance-information (not implemented)
+
+Instead of `expanded:true` we could keep the original expansion information like below, or format it in some other way.
+But that require a bit more work, as today the original expansion information is modfiied during expansion
```

Comment:
This then would be a "mixed" solution?
Means, expanded instances plus original instance information?

Reply:
Yes, some form of mixed solution.